utils/concatAlbumDescription.ts

```diff
@@ -0,0 +1,82 @@
+import { Album } from "@/schemas/collections.schemas";
+
+export default function concatAlbumDescription(album: Album) {
+  if (album.formats.length === 1) {
+    return album.formats
+      .map((format) => {
+        const { name, text, descriptions } = format;
+        const qty = parseInt(format.qty);
+        if (text && descriptions) {
+          if (qty === 1) {
```

Comment:
you can combine these into one return:

```ts
return `${text} ${name} ${descriptions?.join(" ")} ${qty ? `(x${qty})` : ''}`
```

Reply:
probably can use a similar strategy to condense a lot of this code, this is a lot of if statements haha

Reply:
I refactored to have ternaries in each of my returns, not sure how else to narrow it down. If you have any further hints or suggestions let me know.